{"info": {"author": "Alexander Litvinenko", "author_email": "litvinenko1706@gmail.com", "bugtrack_url": null, "classifiers": ["Development Status :: 4 - Beta", "Environment :: Web Environment", "Intended Audience :: Developers", "License :: OSI Approved :: MIT License", "Programming Language :: Python :: 2", "Programming Language :: Python :: 2.7", "Programming Language :: Python :: Implementation :: CPython", "Topic :: Database"], "description": "UNKNOWN", "description_content_type": null, "docs_url": null, "download_url": "UNKNOWN", "downloads": {"last_day": -1, "last_month": -1, "last_week": -1}, "home_page": "https://github.com/absent1706/unittest_onerror", "keywords": "unittest", "license": "MIT", "maintainer": null, "maintainer_email": null, "name": "unittest_onerror", "package_url": "https://pypi.org/project/unittest_onerror/", "platform": "any", "project_url": "https://pypi.org/project/unittest_onerror/", "project_urls": {"Download": "UNKNOWN", "Homepage": "https://github.com/absent1706/unittest_onerror"}, "release_url": "https://pypi.org/project/unittest_onerror/0.1/", "requires_dist": null, "requires_python": null, "summary": "Declarative error and failure catching for Python unittest module", "version": "0.1", "yanked": false, "html_description": "<div class=\"project-description\">\n            <p>UNKNOWN</p>\n\n          </div>"}, "last_serial": 2848704, "releases": {"0.1": [{"comment_text": "", "digests": {"md5": "91ba7e62cfbd63bda212635a9e16c5a5", "sha256": "343c8ad2fcb379d2371e2c1d3b4d7ba03df922ee05bbae1783384507e38a7851"}, "downloads": -1, "filename": "unittest_onerror-0.1.tar.gz", "has_sig": false, "md5_digest": "91ba7e62cfbd63bda212635a9e16c5a5", "packagetype": "sdist", "python_version": "source", "requires_python": null, "size": 1803, "upload_time": "2017-05-03T10:13:34", "upload_time_iso_8601": "2017-05-03T10:13:34.641582Z", "url": "https://files.pythonhosted.org/packages/1b/d9/e8224ceef2fba50f009bfe6b9798cad13a5318ca392c8978f86ab07d4b34/unittest_onerror-0.1.tar.gz", "yanked": false}]}, "urls": [{"comment_text": "", "digests": {"md5": "91ba7e62cfbd63bda212635a9e16c5a5", "sha256": "343c8ad2fcb379d2371e2c1d3b4d7ba03df922ee05bbae1783384507e38a7851"}, "downloads": -1, "filename": "unittest_onerror-0.1.tar.gz", "has_sig": false, "md5_digest": "91ba7e62cfbd63bda212635a9e16c5a5", "packagetype": "sdist", "python_version": "source", "requires_python": null, "size": 1803, "upload_time": "2017-05-03T10:13:34", "upload_time_iso_8601": "2017-05-03T10:13:34.641582Z", "url": "https://files.pythonhosted.org/packages/1b/d9/e8224ceef2fba50f009bfe6b9798cad13a5318ca392c8978f86ab07d4b34/unittest_onerror-0.1.tar.gz", "yanked": false}], "timestamp": "Fri May  8 03:40:19 2020"}